.lando.yml

```diff
@@ -63,6 +80,26 @@ tooling:
 #    dir: "/app/web/themes/custom/vote_gov"
 #    cmd: npm run build --prefix  /app/web/themes/custom/vote_gov
   # Custom lando command, to sync Drupal db with updates, config changes in codebase.
+  'cy:axe':
+    service: node
+    cmd: cd testing && npx cypress run --spec 'cypress/e2e/accessibility/**/*' 
+    description: run cypress accessibility test - headless 
+  'cy:links':
+    service: node
+    cmd: cd testing && npx cypress run --spec 'cypress/e2e/internalLinks/**/*' 
+    description: run cypress internal links tess - headless    
+  'cy:proofer':
+    service: node
+    cmd: cd testing && npx cypress run --spec 'cypress/e2e/externalLinks/**/*' 
```

Comment:
rewrite this line like so:
`cmd: npm run cy:proofer --prefix  /app/testing`

Reply:
updated format